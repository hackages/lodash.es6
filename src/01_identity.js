/**
 ** This method returns the first argument it receives.
 **/
export function identity() {}
