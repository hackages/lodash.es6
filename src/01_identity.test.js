import {identity} from './01_identity'

describe('identity', () => {
  it('Should return the same value that has been passed', () => {
    expect(identity(1)).toMatchSnapshot()
    expect(identity({})).toMatchSnapshot()
    expect(identity('Hackages')).toMatchSnapshot()
    expect(identity(1, 2, 3)).toMatchSnapshot()
  })
})
