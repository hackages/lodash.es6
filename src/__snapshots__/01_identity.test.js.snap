// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`identity Should return the same value that has been passed 1`] = `1`;

exports[`identity Should return the same value that has been passed 2`] = `Object {}`;

exports[`identity Should return the same value that has been passed 3`] = `"Hackages"`;

exports[`identity Should return the same value that has been passed 4`] = `1`;
